applications/new_schema/types.py

```diff
@@ -87,3 +95,59 @@ def resolve_harbor_choices(self, info, **kwargs):
     @view_permission_required(BerthApplication, BerthLease, CustomerProfile)
     def get_node(cls, info, id):
         return super().get_node(info, id)
+
+
+class WinterStorageAreaChoiceType(DjangoObjectType):
+    winter_storage_area = graphene.String(required=True)
```

Comment:
wait, but you said that this causes errors 😅

Reply:
and I said, that "yes, servicemap id are not there, so we can keep this non-required" 🙂

Reply:
when re-testing it,  I accidentally committed it. good catch, will remove :D